src/Symfony/Component/Routing/Attribute/Route.php

```diff
@@ -201,6 +207,28 @@ public function getEnv(): ?string
     {
         return $this->env;
     }
+
+    public function getAliases(): array
+    {
+        return $this->aliases;
+    }
+
+    /**
+     * @param list<string>|string $aliases
+     *
+     * @throws InvalidArgumentException if the "alias" argument is not a string or an array of strings
+     */
+    public function setAliases(array|string $aliases): void
+    {
+        if (\is_array($aliases)) {
+            foreach ($aliases as $a) {
+                if (!\is_string($a)) {
```

Comment:
I don't think we need to be that defensive here. If we have the appropriate `string[]` type in phpdoc, we always consider it as being part of our contract and we generally don't add associated runtime checks (which require looping over the array). Projects wanting extra guarantees can use static analysis to get those benefits.